### ISSUE TITLE
[FEAT] Support for Keyboard Shortcut Keys

### ISSUE THREAD

**Issue by hj-collab** [PhilippC/keepass2android#2691]
Similiar request for KeePassDX. https://github.com/Kunzisoft/KeePassDX/issues/1530

**Comment by MrBreaker666** [comment]
Indeed this is an most needed feature... With the use of an physical keyboard no onscreen keyboard is shown and so there's no way to use the autofill service.

The only way around this issue is to enable the onscreen keyboard while an physical keyboard is connected (not preferred). 

So the option of a "hotkey" and/or keyboard shortcut(s), like the Windows keepass application is much needed => 

Command	Hot Key
Global Auto-Type	Definable in 'Tools' → 'Options' → tab 'Integration'; Ctrl+Alt+A by default.
Global Auto-Type - Password Only	Definable in 'Tools' → 'Options' → tab 'Integration'; Ctrl+Alt+Shift+A by default.
Auto-Type Selected Entry	Definable in 'Tools' → 'Options' → tab 'Integration'; undefined by default.
Show KeePass Window	Definable in 'Tools' → 'Options' → tab 'Integration'; Ctrl+Alt+K by default.

**Comment by PhilippC** [comment]
you can disable autofill/keyboard integration in the settings

**Comment by MrBreaker666** [comment]
Yes I know, but we would like to use autofill (like the autotype function in the original Windows Keepass application) so assigning an hotkey and/or keyboard shortcut would totally solve the problem.
